.github/workflows/issue-tests.yml

```diff
@@ -28,6 +28,14 @@ jobs:
 
     - name: Setup YugabyteDB
       run: |
+        # Note: Update val for LATEST_STABLE here along with 'src/ybversion/constants.go/LATEST_STABLE' variable
+        if [ "${YB_VERSION}" == "LATEST_STABLE" ]; then
```

Comment:
Can you check if we can use env vars for this?  https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#defining-environment-variables-for-a-single-workflow

Becomes slightly simpler because we don't have to set it explicitly here again.

Reply:
@anee I didn't how you are suggesting to use the env var.
Can you share a sample of it

Reply:
@makalaaneesh  I didn't how you are suggesting to use the env var.
Can you share a sample of it